chef/cookbooks/crowbar/recipes/prepare-upgrade-scripts.rb

```diff
@@ -68,3 +70,21 @@
     new_alias: new_alias
   )
 end
+
+# This script shuts down non-essential services on the nodes
+# It leaves only database (so we can create a dump of it)
+# and services necessary for managing network traffic of running instances.
+
+# Find out now if we have HA setup and pass that info to the script
+use_ha = node["run_list_map"].key? "pacemaker-cluster-member"
+
+template "/usr/sbin/crowbar-shutdown-services-before-upgrade.sh" do
```

Comment:
This looks duplicated with https://github.com/jsuchome/crowbar-core/blob/d6a4758f6a65dd40b104df90bce9eb2e6bd3c3a1/chef/cookbooks/crowbar/recipes/prepare-services-shutdown.rb#L25. What's the difference?